File: app/src/main/java/com/example/aerosports/customview/TextViewCSRegular.java
package com.example.aerosports.customview;

import android.content.Context;
import android.util.AttributeSet;

import androidx.appcompat.widget.AppCompatTextView;

import com.example.aerosports.utils.StringUtils;

/**
 * Created by PPCLink on 22/03/2018.
 */

public class TextViewCSRegular extends AppCompatTextView {

    public TextViewCSRegular(Context context) {
        super(context);
    }

    public TextViewCSRegular(Context context, AttributeSet attrs) {
        super(context, attrs);
        setTypeface(StringUtils.setCustomFont(context, "fonts/S-Core - CoreSansGS45Regular1.ttf"));
    }

    public TextViewCSRegular(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
        setTypeface(StringUtils.setCustomFont(context, "fonts/S-Core - CoreSansGS45Regular1.ttf"));
    }

}

File: app/src/main/java/com/example/aerosports/customview/TextViewCSRegular.kt
package com.example.aerosports.customview

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatTextView
import com.example.aerosports.utils.StringUtils
import com.example.aerosports.utils.StringUtils.Companion.setCustomFont

class TextViewCSRegular: AppCompatTextView{
    constructor(context: Context) : this(context, null)
    constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
        typeface = setCustomFont(context, "fonts/S-Core - CoreSansGS45Regular1.ttf")
    }
}